config.py

```diff
@@ -16,3 +15,39 @@
 # This dict holds aspect ratio data for hydrograph rendered in static/js/hydrograph.js. Keys are 'height' and 'width'
 HYDRO_META = {}
 
+# Map configuration
+
+SPATIAL_REFERENCE_ENDPOINT = 'http://spatialreference.org/ref/epsg/${epsg_code}/proj4/'
+PROJECTION_EPSG_CODE = '4326'
+
+#
+BOUNDING_BOX = [-94.294625, 41.015733, -91.424186, 43.102420]
+
+BACKGROUND_FILE = 'floodviz/static/geojson/states.json'
+REFERENCE_FILE = 'floodviz/static/geojson/reference.json'
```

Comment:
reference.json comes from the example reference locations [attached to FV-11](https://internal.cida.usgs.gov/jira/secure/attachment/21203/flood_cities_01.json). I think it would be better to leave it as a file because it might be very large, but I think moving the file to `instance/` makes sense.